### PR TITLE
Add comments feature

### PR DESCRIPTION
**Pull request by andonimihai**
Add comments feature

### CODE REVIEW

**Comment by andonimihai** [comment]
@robinmetral I added comments feature on business/merchant page, can you have a look please? :)

File: pages/business/[id].tsx [hunk 1]
```diff
@@ -1,108 +1,83 @@
...
 
+  const handleFormSubmit = async (e: React.SyntheticEvent) => {
+    e.preventDefault();
+    if (!comment.trim().length) {
+      return;
+    }
+
+    const commentId = new Date().getTime();
+
+    await firebase
```
Comment (robinmetral) [inline pages/business/[id].tsx:53]: Ah so we're doing the firebase DB calls inline again 😅 #hackweek

File: pages/business/[id].tsx [hunk 2]
```diff
@@ -124,6 +99,7 @@ const BusinessPage = ({ merchant, id }: { merchant: Merchant; id: string }) => {
 
   const numberOfClaps = claps[id]?.length;
   const hasClapped = !!claps[id]?.find((clap) => clap.userId === AuthUser.id);
+  // const comments =
```
Comment (robinmetral) [inline pages/business/[id].tsx:102]: 👀

File: src/pages/business/business-page.styles.tsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
+import { Card, SubHeading, Button, Image } from '@sumup/circuit-ui';
```
Comment (robinmetral) [inline src/pages/business/business-page.styles.tsx:1]: Again this is hack week, so no problem, but just FYI for "real" work stuff in the future: 

https://github.com/sumup/ze-dashboard/pull/4530#discussion_r675393786

**Review by robinmetral — APPROVED** [review]
🚀 YOLO 🚀

File: src/pages/business/business-page.styles.tsx (hunk 3 above)
Comment (andonimihai) [reply src/pages/business/business-page.styles.tsx:1]: Thank you so much for this insight. I didn't know about this.
Very useful piece of information!!!! ❤️ 💪

Comment (robinmetral) [reply src/pages/business/business-page.styles.tsx:1]: "Hack week is for learning" – @jmaslate

Comment (robinmetral) [reply src/pages/business/business-page.styles.tsx:1]: 😄

File: pages/business/[id].tsx (hunk 1 above)
Comment (andonimihai) [reply pages/business/[id].tsx:53]: yes :D :D 
I even think this is how it should work. It seems like it's safe to use the client only in front-end apps and `firebase-admin` for servers 😄

Comment (robinmetral) [reply pages/business/[id].tsx:53]: yeah it's totally fine, I just like the idea of handling firebase init and API calls only from API routes, but 🤷 doesn't matter that much

**Comment by andonimihai** [comment]
merged. i think now you can also see comments thing on your machine ))

**Pull request merged**